commands/issues.py

```diff
@@ -0,0 +1,20 @@
+from telegram import InlineKeyboardButton, InlineKeyboardMarkup
+
+
+def issues(update, context):
+    chat_id = update.message.chat_id
+    buttons = [
+        [
+            InlineKeyboardButton(
+                text="Open Issues", url="https://github.com/JIITODC/bot/issues?q=is%3Aopen+is%3Aissue"
+            ),
+            InlineKeyboardButton(
+                text="Closed Issues", url="https://github.com/JIITODC/bot/issues?q=is%3Aissue+is%3Aclosed"
+            ),
```

Comment:
I don't think closed issues matter. You can remove it.

Reply:
removed.